adapters/repos/db/vector/hnsw/condensor_integration_test.go

```diff
@@ -568,7 +569,11 @@ func TestCondensorWithoutEntrypoint(t *testing.T) {
 		require.Nil(t, err)
 
 		bufr := bufio.NewReader(fd)
-		res, _, err := NewDeserializer(logger).Do(bufr, &initialState, false)
+
+		reader := commitlog.NewNoopReaderHasher(bufr)
```

Comment:
Not sure if I missed it but is there a test that tests the whole serialization/deserialization (not noop)?

Reply:
integration tests at `condensor_integration_test.go` are now fully making use of it